language/move-vm/types/src/values/values_impl.rs

```diff
@@ -1774,6 +1775,18 @@ impl Vector {
         self.unpack(type_param, 0)?;
         Ok(())
     }
+
+    pub fn to_blob(self) -> PartialVMResult<Vec<u8>> {
```

Comment:
Why is this change needed?

Reply:
Some functions are here because they are needed for #142 but in fact that one is not longer needed, so I removed it.